### ISSUE TITLE
Command line parsing error with Python 3.8

### ISSUE THREAD

**Issue by thorstenkampe** [pytest-dev/pytest#5523]
- [X] a detailed description of the bug or suggestion
Command line parsing error with Python 3.8 when concatenating short options
- [X] pytest and operating system versions
Pytest 5.0.0, Python 3.8.0b1, Windows 10
- [X] minimal example if possible
```
> pytest -vx
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: -vx
  inifile: None
  rootdir: F:\Documents\Scripts\Runscript
```
`pytest -v -x` works

**Comment by nicoddemus** [comment]
Hmmm indeed we only apply a workaround for that Python bug in Python < 3.8:

https://github.com/pytest-dev/pytest/blob/4bc041572022624bbda81e46efb419fabd8d782e/src/_pytest/config/argparsing.py#L361

@Zac-HD any thoughts? Perhaps we should use `< 3.9` until the patch lands and a new 3.8 gets released?

**Comment by Zac-HD** [comment]
Sure - we should probably drop in an assertion that we're running on Python < 3.9 too, with a message that will remind us what to follow up on if it's slipped until then :smile:

**Comment by nicoddemus** [comment]
Do you have some time to get to this later @Zac-HD? If you don't I can pick it up later in the weekend.

**Comment by Zac-HD** [comment]
Probably not sorry - family commitments all Sunday and a lot of writing I'm already behind on...

**Comment by nicoddemus** [comment]
No worries, thanks!

**Comment by asottile** [comment]
going to unpin since 5.0.1 is out 👍

**Comment by encukou** [comment]
The Python fix should get into the upcoming 3.8 beta 3.
